mirascope/chat/models/openai_chat.py

```diff
@@ -20,7 +20,21 @@
 
 
 class OpenAIChat:
-    """A convenience wrapper for the OpenAI Chat client."""
+    """A convenience wrapper for the OpenAI Chat client.
+
+    The Mirascope convenience wrapper for OpenAI provides a more user-friendly interface
+    for interacting with their API. For detailed usage instructions, check out the
+    following links.
```

Comment:
Include a python example here instead of links to the cookbook (even if it's a direct copy of the cookbook. This is so you don't have to leave the code to see the example in your editor).